opal-r/src/main/java/org/obiba/opal/web/r/Dtos.java

```diff
@@ -54,4 +60,33 @@ private Dtos() {}
         .build();
   }
 
+
+  public static Ws.ClientErrorDto getErrorMessage(Response.StatusType status,
+                                                  REvaluationRuntimeException exception) {
+    String message = exception.getMessage() + ": " + Joiner.on("; ").join(exception.getRMessages());
+
+    return Ws.ClientErrorDto.newBuilder()
+            .setStatus("RServerRuntimeError")
+            .setCode(status.getStatusCode())
+            .addArguments(message.replace("\n", "").replace("\r", ""))
+            .addExtension(OpalR.REvaluationRuntimeErrorDto.errors, OpalR.REvaluationRuntimeErrorDto.newBuilder().build())
+            .build();
+  }
+
+  public static Ws.ClientErrorDto getErrorMessage(Response.StatusType status,
+                                                  RRuntimeException exception) {
+
+    Ws.ClientErrorDto.Builder builder = Ws.ClientErrorDto.newBuilder()
+            .setStatus("RServerRuntimeError")
+            .setCode(status.getStatusCode())
+            .addExtension(OpalR.REvaluationRuntimeErrorDto.errors, OpalR.REvaluationRuntimeErrorDto.newBuilder().build());
```

Comment:
Then a RRuntimeException is wrapped in a REvaluationRuntimeError? I think you mant RRuntimeErrorDto?.